apps/console/src/features/applications/components/wizard/passive-sts-protocol-settings-wizard-form.tsx

```diff
@@ -141,6 +142,45 @@ export const PassiveStsProtocolSettingsWizardForm: FunctionComponent<PassiveStsS
                                 </Hint>
                             </Grid.Column>
                         </Grid.Row>
+                        <Grid.Row>
+                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                <Field
+                                    name="replyToLogout"
+                                    label={
+                                        t("console:develop.features.applications.forms.inboundSTS.fields." +
+                                          "replyToLogout.label")
+                                    }
+                                    required={ true }
```

Comment:
We can do prop shorthand here. No need to define the value explicitly.

Reply:
Addressed with 37333d9